README.md

```diff
@@ -1,52 +1,112 @@
-# Data Diff
+# **data-diff**
 
-**`data-diff` is currently under heavy development, if you run into issues,
+**data-diff is currently under heavy development, if you run into issues,
 please file an issue and we'll help you out ASAP!**
 
-A cross-database, efficient diff using checksums between mostly-similar database
-tables.
-
-- Validate that a table was copied properly
-- Be alerted before your customer finds out, or your report is wrong
-- Validate that your replication mechnism is working correctly
-- Find changes between two versions of the same table
-
-It uses a bisection algorithm and checksums to efficiently check if e.g. a table
-is the same between MySQL and Postgres, or Postgres and Snowflake, or MySQL and
-RDS!
-
-```python
-$ data-diff postgres:/// Original  postgres:/// Original_1diff -t timestamp -v --bisection-factor=4
-[16:57:36] INFO - Diffing tables of size 25000095 and 25000095 | segments: 4, bisection threshold: 1048576.
-[16:58:03] INFO - Diffing segment 1/4 of size 8333364 and 8333364
-[16:58:12] INFO - Diffing segment 2/4 of size 8333365 and 8333365
-[16:58:29] INFO - . Diffing segment 1/4 of size 2777787 and 2777787
-[16:58:32] INFO - . Diffing segment 2/4 of size 2777788 and 2777788
-[16:58:45] INFO - . . Diffing segment 1/4 of size 925925 and 925925
-[16:58:46] INFO - . . Diffing segment 2/4 of size 925929 and 925929
-[16:58:48] INFO - . . . Diff found 2 different rows.
+**data-diff** is a command-line tool and Python library to efficiently diff
+rows across two different databases.
+
+* 🪢 Verifies across [many different databases][1] (e.g. Postgres -> Snowflake)
+* 🔍 Outputs diff of rows in detail
+* 🚨 Simple CLI/API to create monitoring and alerts
+* 🔥 Verify 25M+ rows in less than 10s
+* ♾️  Works for tables with 10s of billions of rows
+
+**data-diff** splits the table into smaller segments, then checksums each
+segment in both databases. When the checksums for a segment aren't equal, it
+will further divide that segment into yet smaller segments, cheksumming those
+until it gets to the offending row(s). See [Technical Explanation][2] for more
```

Comment:
Re "offending", I would stick to something less picturesque, like differing, or changed.